app/components/subcase/subcase-titles-panel/subcase-titles-panel-edit.js

```diff
@@ -47,6 +47,7 @@ export default class SubcaseTitlesPanelEdit extends Component {
     );
     if (this.args.subcase.confidential) {
       yield this.pieceAccessLevelService.updateDecisionsAccessLevelOfSubcase(this.args.subcase);
+      yield this.pieceAccessLevelService.updateSubmissionAccessLevelOfSubcase(this.args.subcase);
     }
```

Comment:
This happens on every change, not just when setting confidential.
Should we listen to the specific change from non-confidential to confidential?

Reply:
Wat we are basically doing is saving the user the trouble of going to the batch edit window to set all access levels manually